src/labeled_posts.json

```diff
@@ -0,0 +1,203 @@
+[
+  {
+    "text": "!WarrenPlanBot student debt",
+    "source": "sh",
```

Comment:
We can use this field to say, for example, this post came from `/r/warren` so that we can, for example, filter posts by source before matching/scoring as desired